src/common/intl.cpp

```diff
@@ -396,6 +396,12 @@ bool wxLocale::DoCommonPostInit(bool success,
             t->AddStdCatalog();
     }
 
+#if defined(__WXOSX__)
+    // LC_CTYPE is set in DoCommonPreInit() (see init.cpp). Set it again here,
```

Comment:
I think we might want have some `wxLocale::OSXFixCRT()` or something like this and call it from both places, even if it's a trivial one-line function because this would allow documenting this issue in a single place -- and hopefully get rid of it easier in the future, too.

Reply:
I've done this now, please let me know if you have any objections.